src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -29,122 +29,140 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/
   - /docs/introduction-new-relic-ai
   - /docs/alerts/new-relic-alerts/getting-started/alerting-new-relic
+  - /docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/
 signupBanner:
   text: Monitor and improve your entire stack. 100GB free. Forever.
 ---
 
-import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.webp'
-
-As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening, where it's happening, and what to do about it. Alerts allow you to define the conditions in your system you want to know about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that proactively notifies your team about potential problems by identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues, and providing a root cause analysis. 
-
-New Relic's alerts and applied intelligence were designed for busy DevOps teams to end response fatigue: you can detect and diagnose incidents and prioritize them for immediate action. 
+import accountsAlertingConcepts from 'images/accounts_diagram_alerting-concepts.webp'
+
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed and New Relic's alerts and applied intelligence are the answer. With alerts, you can configure, detect, and diagnose incidents, prioritize them, and take the needed immediate action. Use alerts to identify anomalies before they reach your end-user. 
```

Comment:
This intro feels like a lot. Maybe it's all the service names? Anyway, I think it could use a screenshot to break it up a little bit or give people something else to look at

Reply:
@paperclypse, I've removed some of the service names and added the main page of alerts. Let me know if it's better now ;-)